libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -278,9 +289,10 @@ export interface IBucketClient {
    * Put an object in the bucket.
    * @param key Key of the object.
    * @param body Content of the object we want to store into the bucket.
+   * @param options Additional options
    * @inflight
    */
-  put(key: string, body: string): Promise<void>;
+  put(key: string, body: string, options?: BucketPutProps): void;
 
```

Comment:
What about the sim implementation? Where are we storing the content type there?

Reply:
Actually, I have no clue. The local filesystem of most OSs doesn't store the Content-Type of a file as metadata since it can infer the type from the file extension. So I'm not sure if Wing Simulator actually needs to handle that.

Reply:
An approach we could use is to create a metadata JSON file with the same name as the uploaded file but with an additional `.meta` extension.

Reply:
I think we will need something like this because we would need a way to query the content type of an object.